avaframe/com1DFA/varyMu.cint

```diff
@@ -61,6 +65,32 @@ void postprocessing(char ResultDir[], char SimName[], float Mu){
     set(name, "set_report_directory", ResultDir_rep);
     set(name, "generate_report", "");
 
+    // Prepare for postprocessing of first and last time step of flow depth parameter
+    sprintf(name2,"Project/Simulations/%s",SimName);
+    sprintf(Sim,"Project/Simulations/%s",SimName);
+    set(Sim, "LoadFirstTimeStep", "");
+
+    //export Flow Depth
+    sprintf(Var,"Project/Simulations/%s/FlowDepth",SimName);
+    set(Var, "Active", "1");
+    sprintf(OutFile,"%s/%s_tFirst_fd.txt",ResultDir_rep, SimName);
+
+    // TODO: make sure same resolution as topo is used
+    set(Var, "ExportRasterSize", "5");
+    set(Var, "ExportFieldRasterAscii", OutFile);
+
+    sprintf(Sim,"Project/Simulations/%s",SimName);
+    set(Sim, "LoadLastTimeStep", "");
+
+
+    //export Flow Depth
+    sprintf(Var,"Project/Simulations/%s/FlowDepth",SimName);
+    set(Var, "Active", "1");
+    sprintf(OutFile,"%s/%s_tLast_fd.txt",ResultDir_rep, SimName);
+
+    // TODO: make sure same resolution as topo is used
+    set(Var, "ExportRasterSize", "5");
+    set(Var, "ExportFieldRasterAscii", OutFile);
```

Comment:
This whole block could go to the generate_report function, right? Since you repeat it 3 times, it's a lot of duplicated code...